echopype/calibrate/cal_params.py

```diff
@@ -165,6 +169,7 @@ def _get_interp_da(
     da_param: Union[None, xr.DataArray],
     freq_center: xr.DataArray,
     alternative: Union[int, float, xr.DataArray],
+    BB_fac: float = 1,
```

Comment:
I suggest just expanding to `BB_factor` for the name to make it more clear. `fac` can get confusing since that can expand to a lot of words 😛

Reply:
Done!